transportable-udfs-avro/src/main/java/com/linkedin/transport/avro/StdUdfWrapper.java

```diff
@@ -23,6 +22,7 @@
 import java.util.List;
 import java.util.stream.IntStream;
 import org.apache.avro.Schema;
+import org.apache.avro.util.Utf8;
```

Comment:
Unused?

Do our builds not fail on unused imports?

Reply:
They do not fail. Fixed.